tensorflow/lite/toco/graph_transformations/ensure_bias_vectors.cc

```diff
@@ -40,8 +41,19 @@ int GetOutputDepthFromWeights(const Model& model, const Operator& op) {
   return 0;
 }
 
+bool CheckOpInputSize(const Operator& op) {
+  if (op.type == OperatorType::kConv ||
+      op.type == OperatorType::kFullyConnected ||
+      op.type == OperatorType::kDepthwiseConv) {
+        return (op.inputs.size() >= 3);
```

Comment:
Can you run the C++ formatter on this PR? https://www.tensorflow.org/community/contribute/code_style#c_coding_style

Reply:
Thanks for review!
I've run them through clang-format, please have a check.

Reply:
I've rebased on master to resolve conflict.